scala-package/infer/src/main/scala/ml/dmlc/mxnet/infer/ImageClassifier.scala

```diff
@@ -195,19 +195,28 @@ object ImageClassifier {
 
   /**
     * Loading input batch of images
-    * @param inputImageDirPath
-    * @return List of buffered images
+    * @param inputImageFiles
+    * @return List of buffered images-
     */
-  def loadInputBatch(inputImageDirPath: String): List[BufferedImage] = {
+  def loadInputBatch(inputImageFiles: List[File]): List[BufferedImage] = {
```

Comment:
take path to files and return a traversable of Buffered Image

check the file exists in the function